ddtrace/contrib/django/patch.py

```diff
@@ -189,12 +189,16 @@ def traced_cache(django, pin, func, instance, args, kwargs):
                 db.ROWCOUNT, sum(1 for doc in result if doc) if result and isinstance(result, Iterable) else 0
             )
         elif command_name == "get":
-            # if valid result and check for special case for Django~3.0 that returns an empty Sentinel object as
-            # missing key
-            if result is not None and result != getattr(instance, "_missing_key", None):
-                span.set_metric(db.ROWCOUNT, 1)
-            # else result is invalid or None, set row count to 0
-            else:
+            try:
```

Comment:
Just to clarify, the reason we need to make this a `try/except` block is that certain cases of Django results are empty sentinel objects that raise errors when applied to `==` conditionals. Am I understanding correctly?

Reply:
Yup